README.md

```diff
@@ -8,19 +8,62 @@
 - Html & CSS
 - Visual studio
 
+## Getting Started
+
+**Follow these instruction below to setup a this project in your local environment**
+
 ### Prerequisites
 
 - Html & CSS basics
 - Git & github
+- SSH key for github linked to git on local environment
```

Comment:
- [ ] [Optional] I think prerequisites of a project are things that you need to install locally before the project will run effectively locally. In that case, the prerequisites of this project should be `node js` and `npm`.😊